src/ZenohUTransport.cpp

```diff
@@ -18,6 +18,9 @@
 
 #include <stdexcept>
 
+#define LOG_LEVEL 0
```

Comment:
This hardcoded level will be removed when we finished tests.
And I am not sure, may be we can move logging to the up-cpp, or use some external logger?

Reply:
`spdlog` is already included in the build for this library. I recommend using that